packages/ses/scripts/hermesTransforms.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { parse } from '@babel/parser';
+import babelGenerate from '@agoric/babel-generator';
+import babelTraverse from '@babel/traverse';
+import * as t from '@babel/types';
+
+// @ts-expect-error TS2339 Property 'default' does not exist
+const traverse = babelTraverse.default || babelTraverse;
+const generate = babelGenerate.default || babelGenerate;
```

Comment:
please add a comment as to why this is needed (maybe https://github.com/Agoric/agoric-sdk/issues/8671?)

Reply:
noted in https://github.com/endojs/endo/pull/2334/commits/0bff9a058663e9ffec09af5ccd085cd191a63429